snippets/csharp/SendOnchain.cs

```diff
@@ -34,6 +34,31 @@ public void MaxReverseSwapAmount(BlockingBreezServices sdk)
         // ANCHOR_END: max-reverse-swap-amount
     }
 
+    public void PreparePayOnchain(BlockingBreezServices sdk, OnchainPaymentLimitsResponse currentLimits, uint feeRate)
+    {
+        // ANCHOR: prepare-pay-onchain
+        var amountSat = currentLimits.minSat;
+        var claimTxFeerate = feeRate;
+
+        try
+        {
+            var resp = sdk.PrepareOnchainPayment(
+                new PrepareOnchainPaymentRequest(
+                    amountSat,
+                    SwapAmountType.SEND,
+                    claimTxFeerate));
+
+            Console.WriteLine($"Send amount, in sats: {resp.senderAmountSat}");
+            Console.WriteLine($"Receive amount, in sats: {resp.recipientAmountSat}");
```

Comment:
Let's use Sender/Recipient amount on all the print statements.

Reply:
Done: 6beebf7c40e86bc2083e79e994ce68990d1c5708